api/builtins_qlik/SearchReplace.go

```diff
@@ -97,6 +101,8 @@ func (p *SearchReplacePlugin) Transform(m resmap.ResMap) error {
 			} else {
 				p.Replace = strings.TrimPrefix(gitVersionTag, "v")
 			}
+		} else if len(p.ReplaceEnvVar) > 0 {
+			p.Replace = os.Getenv(p.ReplaceEnvVar)
```

Comment:
should print something if env value not found ???

Reply:
could log something but logging is rarely turned on so if its blank its agood indiication that the env var was not found